modules/cluster-logging-elasticsearch-scaledown.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-elasticsearch.adoc
+
+[id="cluster-logging-elasticsearch-scaledown_{context}"]
+= Scaling down Elasticsearch nodes
+
+Reducing the number of Elasticsearch nodes in your cluster can result in data loss or performance degradation of the Elasticsearch cluster. 
+
+If you decide to scale down, you should scale by one node at a time and allow the cluster to re-balance the shards and replicas. After the Elasticsearch cluster returns to green status, you can scale down by another node. 
+
+In order to maintain the expected data redundancy, scale down as follows based on the redundancy policy configured for your cluster:
+
+* SingleRedundancy: You can scale down your Elasticsearch nodes to 1.
+* MultipleRedundancy: You can scale down your Elasticsearch nodes to N-1)/2, where N is the number of data nodes.
+* FullRedundancy: You can scale down your Elasticsearch nodes to N-1, where N is the number of data nodes.
+* ZeroRedundancy:   You should not scale down your Elasticsearch nodes. 
```

Comment:
Can we change the order of this to be "zero", "single", "multiple", "full"?

Reply:
this is also misleading... the formulas you have are based on the number of replicas we calculate based on the number of data nodes. it is not equivalent to how far down a cluster can be scaled. This would be better thought of as the rate you _could_ safely scale down because of the chance of either the primary or replica shard being on the node that is removed from the cluster. I think we should be cautious about stating what customers can scale down to since any time they scale down they increase the load on remaining cluster nodes.
Zero - unable to scale down (rate is 0)
Single - scale down one at a time
Multiple - can scale down (n-1)/2 at a time
Full - can scale down any number

Reply:
@ewolinetz What if we get rid of the list? Scaling down by any number higher than one seems to run contrary to the advice to scale down one node and wait for ES to return to green.
Just a note instead?
[NOTE]
====
If your Elasticsearch cluster is set to ZeroRedundancy, you should not scale down your Elasticsearch nodes. 
====